README.md

```diff
@@ -1,6 +1,4 @@
-[![Build Status](https://travis-ci.org/slaclab/pydm.svg?branch=master)](https://travis-ci.org/slaclab/pydm) 
-[![Build status](https://ci.appveyor.com/api/projects/status/sw8rp4an7o79m833?svg=true)](https://ci.appveyor.com/project/hhslepicka/pydm-fxpsx)
-[![Code Health](https://landscape.io/github/slaclab/pydm/master/landscape.svg?style=flat)](https://landscape.io/github/slaclab/pydm/master) 
+[![Build Status](https://dev.azure.com/hhslepicka/pydm-test/_apis/build/status/PyDM%20Pipeline?branchName=master)](https://dev.azure.com/hhslepicka/pydm-test/_build/latest?definitionId=2&branchName=master)
```

Comment:
`https://dev.azure.com/hhslepicka` - should the pydm stuff move to a new azure organization?

Reply:
Hummmm maybe I copied the wrong link. Let me check

Reply:
Great catch! I had both orgs open and I ended up copying from the wrong one.